transport/http2_server.go

```diff
@@ -163,6 +165,11 @@ func (t *http2Server) operateHeaders(hDec *hpackDecoder, s *Stream, frame header
 		return s
 	}
 	t.mu.Lock()
+	if t.goaway {
+		//Stop creating streams on this transport
```

Comment:
Thanks. Let me know if you have problem to understand the code. Or if it is too hard, you can leave it to me. :)